src/utils/file.c

```diff
@@ -1094,6 +1306,32 @@ typedef struct fioGZFile
 	Bytef    buf[ZLIB_BUFFER_SIZE];
 } fioGZFile;
 
+/* check if remote agent encountered any error during execution of async operations */
+int
+fio_check_error_fd_gz(gzFile f, char **errmsg)
+{
+	if (f && ((size_t)f & FIO_GZ_REMOTE_MARKER))
```

Comment:
Could you please add a comment to FIO_GZ_REMOTE_MARKER definition?

Reply:
Sure